controllers/chat/getChannelDetail.js

```diff
@@ -1,3 +1,4 @@
+const moment = require('moment');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `moment` library is a good choice for handling date and time manipulations. However, it's important to ensure that this new dependency is added to your `package.json` file. If not, this could lead to issues when deploying or sharing your code with others as they might not have this package installed.

```diff
+ "dependencies": {
+    "moment": "^2.29.1"
+  }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->